src/net_processing.h

```diff
@@ -61,4 +61,17 @@ bool GetNodeStateStats(NodeId nodeid, CNodeStateStats &stats);
 /** Increase a node's misbehavior score. */
 void Misbehaving(NodeId nodeid, int howmuch);
 
+/** Process protocol messages received from a given node */
+bool ProcessMessages(CNode* pfrom, CConnman& connman, const std::atomic<bool>& interrupt);
+/**
+ * Send queued protocol messages to be sent to a give node.
+ *
+ * @param[in]   pto             The node which we are sending messages to.
+ * @param[in]   connman         The connection manager for that node.
+ * @param[in]   interrupt       Interrupt condition for processing threads
+ * @return                      True if there is more work to be done
+ */
+bool SendMessages(CNode* pto, CConnman& connman, const std::atomic<bool>& interrupt);
```

Comment:
Looks like a rebase error - these functions were moved to the PeerLogicValidation.